config/routes.rb

```diff
@@ -27,6 +27,11 @@
     end
   end
 
-  resources :issues, only: [:new, :create, :show]
+  resources :issues, only: [:new, :create, :show] do
+    resources :comments, module: :issues, only: [:create, :destroy]
```

Comment:
Try to path minimum amount parameters through url, especially if some of them is not necessary.